urnai/actions/action_base.py

```diff
@@ -0,0 +1,20 @@
+from abc import ABC, abstractmethod
+
+
+class ActionBase(ABC):
+	id = None
```

Comment:
An `__init__` method to assign the value of `id` would be nice.

Reply:
I'm confused, you mentioned something else in a previous comment. Its the first change requested on March 9th, i set it to unsolved.